src/Symfony/Bridge/Doctrine/Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -22,6 +22,9 @@
 use Symfony\Component\VarDumper\Cloner\Data;
 use Symfony\Component\VarDumper\Dumper\CliDumper;
 
+// Doctrine DBAL 2 compatibility
+class_exists('Doctrine\DBAL\Platforms\MySqlPlatform');
```

Comment:
why not follow fabbot suggestion to use ::class? this is used alot in such cases already

Reply:
Fair enough, changed.